plugins/samson_ledger/test/config/initializers/samson_ledger/client_test.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+require_relative "../../../test_helper"
+
+SingleCov.covered! file: 'plugins/samson_ledger/config/initializers/samson_ledger/client.rb'
+
+describe 'Ledger::Client' do
+  let(:deploy) { Deploy.first }
+  before do
+    stub_request(:post, "https://foo.bar/api/v1/events")
+  end
+  after do
+    ENV.delete("LEDGER_BASE_URL")
+    ENV.delete("LEDGER_TOKEN")
+  end
+
+  describe ".plugin_enabled?" do
+    around do |test|
+      ENV["LEDGER_BASE_URL"] = 'https://foo.bar'
```

Comment:
ideally move this to the top instead of repeating it, should be the default for all these tests ...
.. and use with_env

```
describe "foo" do
  with_env(token: 'xyz')
  it "bar" do
  end
end
```